contracts/core/liquidators/twap-impl/TWAPAuction.sol

```diff
@@ -157,18 +166,20 @@ contract TWAPAuction is TwoAssetPriceBoundedLinearAuction {
             _liquidatorData.chunkSizeValue
         );
 
-        // Initialize first chunk auction
-        LinearAuction.initializeLinearAuction(
-            _twapAuction.chunkAuction,
-            _currentSet,
-            _nextSet,
-            chunkSize
-        );
+        // Normalize the chunkSize and orderSize to ensure all values are a multiple of 
+        // the minimum bid
+        uint256 minBid = _twapAuction.chunkAuction.auction.minimumBid;
+        uint256 normalizedChunkSize = chunkSize.div(minBid).mul(minBid);
```

Comment:
So when we validate the twap parameters we check to make sure that the length of the auction won't exceed the rebalanceAuctionPeriod. Unfortunately that's done before any of this rounding and as a consequence we could under estimate the number of auction iterations required to complete the auction. So we could in theory allow an auction that we might expect to exceed the rebalanceAuctionPeriod.

Obviously that is very much an estimate and it's definitely an extremely narrow edge case but it does exist. Don't know if maybe it makes sense to move that particular validation directly into this function instead of having it validated before initialization takes place.

Reply:
We could move the check after, I suppose. It doesn't feel like an issue, but in what mathematical issues could this be a problem? We could increase the buffer